### ISSUE TITLE
[VFP] Leavening agent has no recipe

### ISSUE THREAD

**Issue by hron84** [Wabbit0101/mods_hoardercraft#41]
In "Life in village" modpack I have VanillaFoodPantry 4.1.0 but it has no recipe. Thus, I cannot craft a dough.

I've an other way to make a normal bread and skip your custom ones, but is there any way to disable vanilla recipe overriding and let 3 wheat make a bread?

**Comment by Wabbit0101** [comment]
I think something else is altering the recipe registry. As a strict rule, VFP does not alter vanilla items or recipes as it's intended as a supplement to those. Attach your mod list and VFP config to issue for me to check. Alternately you can try something like NoMoreRecipeConflict to find out what's in conflict in your modpack...

**Comment by hron84** [comment]
```
AppleCore-mc1.12.2-3.1.1.jar
AppleSkin-mc1.12-1.0.9.jar
astralsorcery-1.12.2-1.8.10.jar
AutoRegLib-1.3-17.jar
Baubles-1.12-1.5.2.jar
BetterAdvancements-1.12.2-0.0.5.17.jar
BetterBuildersWands-1.12-0.11.1.245+69d0d70.jar
BetterFoliage-MC1.12-2.1.10.jar
BetterQuesting-3.5.256.jar
BetterWithMods-1.12-2.1.16.jar
BiblioCraft[v2.4.3][MC1.12.0].jar
Biome_Bundle-1.12.2-v6.1.jar
BirdsNests1.12.2-2.1.0.jar
Blur-1.0.4-14.jar
Chameleon-1.12-4.1.3.jar
ChestTransporter-1.12.2-2.8.8.jar
Chisel-MC1.12.2-0.2.0.31.jar
Controlling-3.0.6.jar
CookingForBlockheads_1.12.2-6.3.26.jar
CoralReef-2.5-1.12.2.jar
CraftTweaker2-1.12-4.1.8.jar
CTM-MC1.12-0.2.3.12.jar
cucumber-1.12-1.1.0.jar
CustomMainMenu-MC1.12.2-2.0.8.jar
dailies-1.12.2-6.jar
DungeonTactics-1.12.2-0.14.2.jar
DynamicSurroundings-1.12.2-3.4.9.13.jar
extraalchemy-0.3.8.4-release.jar
FamiliarFauna-1.12.2-1.0.10.jar
Farseek-1.12-2.2.5.jar
FastLeafDecay-v14.jar
foamfix-0.9.9.1-1.12.2-anarchy.jar
GardenStuff-1.12-2.1.1.jar
Hwyla-1.8.25-B40_1.12.jar
ImprovedHoes-1.12-1.2.2.jar
InGameInfoXML-1.12.2-2.8.2.94-universal.jar
Inspirations-1.12.2-0.1.4.jar
InventoryTweaks-1.63.jar
IronBackpacks-1.12.2-3.0.7-10.jar
jei_1.12.2-4.9.1.187.jar
journeymap-1.12.2-5.5.2.jar
JustAFewFish-1.7_for_1.12.jar
justenoughbuttons-1.12.2-1.2.2-5.jar
JustEnoughResources-1.12.2-0.8.6.28.jar
LagGoggles-FORGE-1.12.2-4.1.jar
landlust-1.12.2-0.3.6.jar
Level Up! 2-1.1.19-1.12.jar
LunatriusCore-1.12.2-1.2.0.42-universal.jar
Mantle-1.12-1.3.2.24.jar
megaloot-1.12.2-0.1.4.jar
millenaire-6.2.1.jar
minecolonies-universal-1.12.2-0.8.7582.jar
mobends-0.24_for_MC-1.12.jar
modtweaker-4.0.12.jar
MouseTweaks-2.8-mc1.12.1.jar
MTLib-3.0.4.jar
Neat 1.4-15.jar
NetherEx-1.12-2.1.14.15.jar
NoMobSpawningOnTrees-1.2.1-mc1.12.jar
Notes-1.12-1.1.1.jar
Nutrition-1.12.2-3.2.0.jar
OpenTerrainGenerator-1.12.2 - v6.jar
pickletweaks-1.12-2.0.16.jar
pinklysheep-mc1.12-4.0b4.jar
PiTweaks-1.12-0.5.5.jar
Placebo-1.12.2-1.3.4.jar
PotionCore-1.6_for_1.12.2.jar
prefab-1.3.0.20.jar
primitivemobs-1.1.6_beta.jar
QualityTools-1.0.6_for_1.12.2.jar
Quark-r1.4-123.jar
questbook-3.1.1-1.12.jar
RealBench-1.12.2-1.3.3.jar
Repurpose-1.0.0.17.jar
ResourceLoader-MC1.12.1-1.5.3.jar
RoguelikeDungeons-1.12.2-1.8.0.jar
rustic-1.0.4.jar
solcarrot-1.12.2-1.3.1.jar
SoundFilters-0.10_for_1.12.jar
StandardExpansion-3.4.141.jar
StorageDrawers-1.12.2-5.3.6.jar
Streams-1.12-0.4.3.jar
terracartreloaded-1.12.2-1.2.3.jar
Thaumcraft-1.12.2-6.1.BETA13.jar
ThaumicInventoryScanning_1.12.2-2.0.10.jar
ThaumicJEI-1.12.2-1.2.3-7.jar
TombManyGraves-1.12-4.0.3.jar
uniquecrops-1.12-0.1.52.jar
vanillafoodpantry-mc1.12.1-4.1.0.jar
VanillaTweaks-1.3.1.jar
VeinMiner-1.12-0.38.1.639+134fb1e.jar
ViesCraft-1.12.2-5.9.10.jar
VTweaks-1.12.2-1.4.12b.jar
WailaHarvestability-mc1.12-1.1.12.jar
Wawla-1.12.2-2.5.253.jar
Waystones_1.12.2-4.0.43.jar
XaerosWorldMap_1.2.4_Forge_1.12.jar
```

```
# Configuration file

~CONFIG_VERSION: 1

"a options" {
    # Set to false to disable generation of experience from crafting VFP foods. Note that this option does not affect smelting or mining Xp or game advancements. Enabled by default.
    B:achievements_and_xp_enabled=true

    # Set to true to enable use of potion water bottles in VFP recipes that require a bottle of water. Disabled by default as a crafting table does not differentiate between brewed potions and the plain potion water bottle.
    B:allow_potionitem_as_plain_water=false

    # DEVELOPER. Development messages and tooltips enabled.
    B:debug_mode=false

    # Set to false to disable decker sandwich ingredient (name) tooltips. If enabled you will see the name of each food item that went into your decker sandwich. Enabled by default.
    B:decker_ingredients_tooltip=true

    # Set to false to disable -ALL- custom VFP drops from blocks and mobs unconditionally. Overrides all other settings. OFF by default. Requires restart if changed in-game.
    B:drops_enabled=true

    # Set to false to disable all mod-specific additions to structure and activity loot tables. Changing this affects things like fishing and dungeon loot but NOT upsize drops (see individual options). Enabled by default; requires restart if changed in-game.
    B:enrich_loot_tables=true

    # Set to false to disable compensation for Minecraft ignoring small amounts of crafting Xp. Note that this option is ignored if crafting Xp is disabled. Enabled by default.
    B:generous_xp=true

    # Set to false to disable generation of custom VFP ores like rock salt and natron in new worlds and new chunks. Enabled by default.
    B:ore_generate_enabled=true

    # Set to false to disable registration of and all recipes for VFP's potions like Wolf's Spirit potion. Enabled by default.
    B:potions_enabled=true

    # Set to true to force most basic recipes to have a mod-specific element to ensure no collisions with other mod recipes; for example, instead of just a bucket of water to make bits o' water, the recipe will require a water bucket and a mod-specific item. OFF by default.
    B:super_compatibility_mode=false

    # Set to true to force all VFP items and blocks to a single custom 'VFP' tab in creative views. Disabled by default.
    B:use_custom_tab=false
}


"b bakery" {
    # Set to true to disable the VFP check for a pre-existing flour recipe. Normally VFP will not include its own (1 wheat to 1 flour) recipe if one exists (regardless of what ingredients the other one uses). Disabled by default. Overrides super-compatibility mode.
    B:always_include_simple_flour_recipe=false
}


"c butcher" {
    # Set to true to disable the VFP check for a pre-existing fried egg smelting recipe. Normally VFP will not include its own smelt 1 egg recipe if one exists (regardless of what ingredients or method the other one uses). Disabled by default. Overrides super-compatibility mode.
    B:always_include_fried_egg_recipe=false

    # Set to true to enable smelting of big bones directly to gelatin powder. If disabled you must obtain gelatin using the craft-table recipes. Disabled by default.
    B:enable_bone_smelting=false

    # Set to true to enable repair of butchering axes with standard flint. Disabled by default.
    B:enable_butcheraxe_repair=false

    # Set to true to disable ONLY meat drops from killing guardians when 'include_guardian_drops' is enabled; ignored otherwise. Use if you're interested in tail fins and air sac drops only. Disabled by default.
    B:exclude_guardian_meat=false

    # Set to true to disable ONLY meat drops from killing horses when 'include_horse_drops' is enabled; ignored otherwise. Use if you have another mod that already drops horse meat. Disabled by default.
    B:exclude_horse_meat=false

    # Set to true to disable ONLY meat drops from killing llamas when 'include_llama_drops' is enabled; ignored otherwise. Use if you have another mod that already drops llama meat. Disabled by default.
    B:exclude_llama_meat=false

    # Set to false to ENABLE inclusion of meat drops from killing squid; ignored if 'include_squid_drops' is false. Enable if you do NOT have another mod that already drops squid meat. Note that some VFP recipes require our squid drop and become effectively disabled if this option is off. Enabled by default (no squid meat is dropped).
    B:exclude_squid_meat=true

    # Set to true to disable ONLY meat drops from killing wolves when 'include_wolf_drops' is enabled; ignored otherwise. Use if you have another mod that already drops wolf meat. Disabled by default.
    B:exclude_wolf_meat=false

    # Set to false to disable all VFP drops from killing a bat. VFP adds bat meat, bat poop (used to create rotten flesh or as a fertilizer like bone meal), and special drops for potions. Enabled by default.
    B:include_bat_drops=true

    # Set to false to disable all VFP drops from killing polar bears. VFP adds raw bear meat and special drops for custom potions. Enabled by default.
    B:include_bear_drops=true

    # Set to true to enable VFP bone drops regardless of the other mods present and super-compatibility mode setting. Requires restart if changed in game. [default: *DEFAULT*]
    S:include_bone_drops=*DEFAULT*

    # Set to false to disable stomach lining drops from killing baby cows. VFP adds calf stomach lining for use in creating curdling agent for cheese. Enabled by default.
    B:include_calf_drops=true

    # Set to false to disable all VFP drops from killing guardians and elder guardians. VFP adds guardian tail fins, meat, and other items. Enabled by default.
    B:include_guardian_drops=true

    # Set to false to disable all VFP drops from killing horses. VFP adds drops like horse tears and raw horse meat. Enabled by default.
    B:include_horse_drops=true

    # Set to false to disable all VFP drops from killing llamas. VFP adds raw llama meat and fleece (like wool) blocks. Enabled by default.
    B:include_llama_drops=true

    # Set to false to disable the ability to steal lives from ocelots as Felis Life Essence. Note that this option changes the effect of feeding pufferfish to untamed ocelots! Enabled by default.
    B:include_ocelot_drops=true

    # Set to false to disable all VFP drops from killing pigs. VFP adds drops like raw pork hocks and pig tails. Enabled by default.
    B:include_pig_drops=true

    # Set to false to disable all VFP drops from killing squid. VFP adds raw squid meat and ink pearl shard drops from squid. Enabled by default.
    B:include_squid_drops=true

    # Set to false to disable all VFP drops from killing wolves (tamed and untamed). VFP adds raw wolf meat and special drops for custom potions. Enabled by default.
    B:include_wolf_drops=true

    # Set to true to have llamas drop regular (sheep) wool blocks instead of the custom VFP fleece wool blocks. Disabled by default. Ignored if llama drops in general are disabled.
    B:llamas_drop_regular_wool=false

    # Set to true to increase the number of meat dropped from killing cows, pigs, sheep, etc. The increase is from 0 to 3 more pieces with auxilliary items like hides, feathers, and bones always included. Enabled by default.
    B:upsize_meat_drops=true
}


"d produce" {
    # Set to false to stop acorn drops when you break oak and dark-oak leaves. Enabled by default.
    B:include_acorn_drops=true

    # Set to false to stop raftugli fruit drops when you break jungle leaves. Enabled by default.
    B:include_raftugli_drops=true

    # Set to false to disable additional mushroom drops from chopping down huge mushrooms with a butchering axe or other tool with fortune enchantment. Also adds red mushrooms to mooshroom drops. Enabled by default.
    B:upsize_mushroom_drops=true

    # Set to false to disable (un)crafting of 9 mushrooms from standard mushroom blocks. If enabled you can increase storage size of mushrooms per slot and unlock all the mushroomy goodness of giant mushrooms! Enabled by default.
    B:use_mushroom_blocks_as_storage=true
}


"e upsizer" {
    # Max stack size for all bottled items like fizzy and milk drinks. Default stack size is 24. Allowed sizes are [1,4,8,16,24,32,64].
    I:drinks_max_stacksize=24

    # Max stack size for all just-an-egg items like fried eggs that we inherit from other mods. Preferred stack size is 64; allowed sizes are [16,32,64]. Does NOT affect raw eggs' stack size.
    I:eggs_max_stacksize=64

    # Max stack size for all soupy items like creamy soups, chowders, and stews. Default stack size is 24. Allowed sizes are [1,4,8,16,24,32,64].
    I:soups_max_stacksize=24
}


"x integration" {
    # Set to false to disable integration with automation mods like SmartHoppers that support auto-crafting mechanics. Enabled by default.
    B:automation_enabled=true

    # Set to false to disable recipe-making with non-vanilla foodstuffs (for example Harvestcraft). Note turning this off _after_ you've created mod-dependent items can result in lost items. Enabled by default. Requires game restart.
    B:recipe_use_enabled=true

    rew {
        # ADVANCED. Set to false to disable installing 'food*' dictionary mappings for ReW-based foodstuffs. Only used if RealWorld mod detected and is a version where this tweak is needed (ignored otherwise). Enabled by default.
        B:fix_food_dictionary=true

        # ADVANCED. Set to false to disable installing 'tree*' dictionary mappings for ReW-based leaves, saplings, logs, etc. Only used if RealWorld mod detected and is a version where this tweak is needed (ignored otherwise). Enabled by default.
        B:fix_tree_dictionary=true

        # Set to false to disable crafting recipes for obtaining ReW crops seeds from dropped crops. Enabled by default.
        B:include_seed_recipes=true
    }

    hvc {
        # ADVANCED. Set to false to disable installing 'food*' dictionary mappings for Harvestcraft-based foodstuffs. Only used if Harvestcraft mod detected and is a version where this tweak is needed (ignored otherwise). Enabled by default.
        B:fix_food_dictionary=true
    }

    nat {
        # ADVANCED. Set to false to disable installing 'food*' dictionary mappings for Natura-based foodstuffs. Only used if Natura mod detected and is a version where this tweak is needed (ignored otherwise). Enabled by default.
        B:fix_food_dictionary=true
    }

}


"z miscellaneous" {
    # Set to false to disable red flint drops when not in super-compatibility mode. Some recipes require red flint. If you want these recipes always, leave this option on. Enabled by default.
    B:always_include_red_flint_drops=true

    # Set to false to disable the extra craft recipes for water buckets and bottles (for instance the recipe to convert ice blocks to buckets of water). Enabled by default.
    B:extra_water_items_crafting=true

    # Set to true to limit the use of the flint-cutter to only food RELATED recipes. Disabled by default (cutter can be used to cut things like wool blocks).
    B:flint_cutter_for_food_only=false

    # Set to true to enable the gravel craft recipes (both manual and automated for SmartHoppers). Disabled by default.
    B:gravel_crafting=false

    # DEVELOPER. Set to true to enable extra craft recipes for migration of other food mod crafted items to their equivalent (or mostly equivalent) VFP items. Disabled by default.
    B:pantry_migration_crafting=false

    # Set to false to disable seed drying from source produce. By default you can use seed drying paper to extract seeds from melons, pumpkins, and wheat. Enabled by default; requires restart if changed in-game.
    B:seed_drying=true

    # Set to true to enable feline life essence based craft recipes for certain spawn eggs. Disabled by default; requires restart if changed in-game.
    B:spawnegg_crafting=false

    # Set to true to enable a more complex raw fries recipe that requires salt and a vinegar bath.  Disabled by default.
    B:use_harder_fries_recipe=false
}
```

**Comment by Wabbit0101** [comment]
Betterwithmods' setting of standard vanilla items like mushroom items to NULL via aliasing is causing this (you can disable this in its config under tweaks). I can't "fix" this on VFP's side as you can't have working recipes with references to non-existing items...see issue #43. However, I can give alternate paths to these required items. I will add a way to get leavening and dried seeds in the next version which is to be released within a week. Thanks. (Temporarily disable the option if you can.)

**Comment by hron84** [comment]
@Wabbit0101 it would be great if you can work it around because as I play with a non-own modpack, config files are subject to be replaced as the modpack get updates.

**Comment by Wabbit0101** [comment]
This issue be addressed in the next version which should be out sometime this weekend. These aliases and replacements are starting to trickle into the simpliest mods these days...

**Comment by Wabbit0101** [comment]
Please check but this should be fixed in the 4.2 release. See https://minecraft.curseforge.com/projects/vanillafoodpantry-mod/files/2568445.

**Comment by hron84** [comment]
@Wabbit0101 I'll check it as soon as possible
